### PR TITLE
Update paragraphs.md

### PR DESCRIPTION
**Pull request by reeceatkinson**
### Description
- font-family: system-ui;
- Took out double spaces.

### Type of Change
- Updating the documentation

### Checklist

- [X] All writings are my own.
- [X] My entry follows the Codecademy Docs style guide.
- [X] My changes generate no new warnings.
- [X] I have performed a self-review of my own writing and code.
- [X] I have checked my entry and corrected any misspellings.
- [X] I have made corresponding changes to the documentation if needed.
- [X] I have confirmed my changes are not being pushed from my forked `main` branch.
- [X] I have confirmed that I'm pushing from a new branch named after the changes I'm making.

### CODE REVIEW

**Review by SSwiniarski — APPROVED** [review]
Looks fine as-is.

File: content/html/concepts/paragraphs/paragraphs.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 Title: 'Paragraphs'
-Description: 'Paragraph element  contains and displays a block of text. They are among the most common text-based HTML elements. They are block-level, which means it is as wide as its parent element and as tall as its contents height. The pargraph element is enclosed by an opening tag  and a closing tag : html  This is a block of text! Lorem ipsum dolor sit amet, consectetur adipisicing elit.'
+Description: 'Paragraph element contains and displays a block of text. They are among the most common text-based HTML elements. They are block-level, which means it is as wide as its parent element and as tall as its contents height. The pargraph element is enclosed by an opening tag  and a closing tag : html  This is a block of text! Lorem ipsum dolor sit amet, consectetur adipisicing elit.'
```
Comment (yangc95) [inline content/html/concepts/paragraphs/paragraphs.md:3]:
```suggestion
Description: 'Paragraphs are block-level elements that contain and display a block of text. They are enclosed by an opening and closing tag.'
```

**Review by yangc95 — COMMENTED** [review]
Condensed the description. Also going to go in and make a small change to another part of the entry

**Review by yangc95 — APPROVED** [review]
Looks good to merge!

**Comment by yangc95** [comment]
@reeceatkinson Just realized after merging that we are going to have to change the image below as well to match. Going to do this in a separate PR.

cc: @SSwiniarski

**Comment by yangc95** [comment]
Alright @reeceatkinson, it's live!

https://www.codecademy.com/resources/docs/html/paragraphs

cc: @SSwiniarski

**Comment by reeceatkinson** [comment]
> Alright @reeceatkinson, it's live!
> 
> 
> 
> https://www.codecademy.com/resources/docs/html/paragraphs
> 
> 
> 
> cc: @SSwiniarski 

Thanks Everyone!

**Pull request merged**